sdk/appconfiguration/app-configuration/CHANGELOG.md

```diff
@@ -1,11 +1,18 @@
 # Release History
 
-## 1.3.2 (Unreleased)
+## 1.4.0 (Unreleased)
 
 ### Features Added
 
+- Migrated from using `@azure/core-http` to depend on newer version of Core libraries `@azure/core-client` and `@azure/core-rest-pipeline` which bring better maintainability and performance. [#20766](https://github.com/Azure/azure-sdk-for-js/pull/20766). See [Azure Core v1 vs v2](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/core/core-rest-pipeline/documentation/core2.md) for more on the difference and benefits of the move.
+
 ### Breaking Changes
 
+- As part of the migration to `@azure/core-client` and `@azure/core-rest-pipeline`,
+  - Responses returned from the methods such as `AppConfgurationClient#addConfigurationSetting` no longer expose the `_response` property.
```

Comment:
Yeah, since we documented this in the exported types, we either need a major bump (in which case we can ditch the compat dependency) or we have to wait for Sarangan's work to support `_response` in the shim.

Reply:
> Breaking changes require major version bump?

I thought we weren't doing a major bump from my understanding from service-bus.

> Yeah, since we documented this in the exported types, we either need a major bump (in which case we can ditch the compat dependency) or we have to wait for Sarangan's work to support _response in the shim.

Sounds good either way. However, I should check with @AlexGhiondea on the major bump.